gem_config.rb

```diff
@@ -36,6 +36,7 @@ def self.metadata
 
   def self.dev_dependencies
     [["bundler", ">= 1.3.0"],
+     ["mime-types", ">= 3.0.0"],
```

Comment:
@arunagw made this only a dev dependency minimum as it seems to be only required for ruby 3.
weirdly enough it allows for a later version of jeweler by downgrading a few other gems.

![Screenshot_20210407_063934](https://user-images.githubusercontent.com/98919/113775470-1afd4480-976c-11eb-9f32-440df5a0b7c2.png)

Reply:
maybe worth considering dropping jeweler for the bundler built in gem support

Reply:
I think `jeweler` can be dropped.

Reply:
should I drop it in this PR instead of setting minimum mime-types?

Reply:
Yes. I think it's better to drop unused gems.